### PR TITLE
Vova development

### PR DESCRIPTION
**Pull request by bandvov**
fixed filters button

### CODE REVIEW

File: fn/client/src/components/login/Login.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const Login = ({ storeService, setUserLogged, setUserLoading, userLogged, userLo
     const setInitialCart = (res) => {
         const { accessToken, refreshToken, userId } = res
         const cart = { cartNumbers: 0, cartProducts: [] }
-        axios.put(`https://lv487node-backend.herokuapp.com/users/cart/${userId}`, { cart });
+        axios.put(`http://localhost:5000/users/cart/${userId}`, { cart, "x-auth-token": accessToken });
```
Comment (oleksandrhodzevych) [inline fn/client/src/components/login/Login.js:53]: don use hard code basicUrl
it should be in config!
here use only path without basicUrl

File: fn/client/src/reducers/Cart-reducer.js [hunk 2]
```diff
@@ -5,31 +5,31 @@ const initialState = { cartNumbers: 0, cartProducts: [] }
 const userId = JSON.parse(localStorage.getItem("userId"));
 const productCollection = JSON.parse(localStorage.getItem("products-collection"));
 const localCartNumbers = JSON.parse(localStorage.getItem("cart-numbers"));
+const accessToken = JSON.parse(localStorage.getItem('accessToken'))
 
-const saveCart = async (userId, cart) => {
-  axios.put(`https://lv487node-backend.herokuapp.com/users/cart/${userId}`, { cart });
+const saveCart = async (userId, data, token) => {
+  axios({ method: 'PUT', url: `http://localhost:5000/users/cart/${userId}`, data, headers: { "x-auth-token": token } });
```
Comment (oleksandrhodzevych) [inline fn/client/src/reducers/Cart-reducer.js:11]: hard code basic URL not permit

File: fn/client/src/reducers/Cart-reducer.js [hunk 3]
```diff
@@ -5,31 +5,31 @@ const initialState = { cartNumbers: 0, cartProducts: [] }
...
-      const { cart } = res.data.user
-      if (!cart) {
-        saveCart(userId, { cartNumbers: 0, cartProducts: [] })
-        return
-      }
-      initialState.cartNumbers = cart.cartNumbers
-      initialState.cartProducts = cart.cartProducts
+const setInitial = async (token) => {
+  if (userId) {
+    const res = await axios({ method: 'GET', url: `http://localhost:5000/users/${userId}`, headers: { "x-auth-token": token } })
```
Comment (oleksandrhodzevych) [inline fn/client/src/reducers/Cart-reducer.js:15]: same about basicUrl it should be in config

File: fn/client/src/components/login/Login.js (hunk 1 above)
Comment (bandvov) [reply fn/client/src/components/login/Login.js]: fixed

File: fn/client/src/reducers/Cart-reducer.js (hunk 2 above)
Comment (bandvov) [reply fn/client/src/reducers/Cart-reducer.js]: fixed

File: fn/client/src/reducers/Cart-reducer.js (hunk 3 above)
Comment (bandvov) [reply fn/client/src/reducers/Cart-reducer.js]: fixed

**Pull request merged**